* Make sure it's built before it's used (thanks to Drake Wyrm).


--- indent-2.2.10/man/Makefile.am.orig	2008-03-11 19:50:42.000000000 +0100
+++ indent-2.2.10/man/Makefile.am	2009-11-23 04:11:17.000000000 +0100

@@ -10,6 +10,6 @@
 
 MAINTAINERCLEANFILES = @PACKAGE@.1 Makefile.in texinfo2man 
 
-@PACKAGE@.1: ${srcdir}/@PACKAGE@.1.in  ${srcdir}/../doc/@PACKAGE@.texinfo texinfo2man.c  Makefile.am
+@PACKAGE@.1: ${srcdir}/@PACKAGE@.1.in  ${srcdir}/../doc/@PACKAGE@.texinfo texinfo2man Makefile.am
 	./texinfo2man ${srcdir}/@PACKAGE@.1.in ${srcdir}/../doc/@PACKAGE@.texinfo > $@
 
